src/Lykke.Service.BlockchainSettings.Services/BlockchainValidationService.cs

```diff
@@ -53,7 +53,7 @@ public async Task<bool> ValidateServiceUrlAsync(string serviceUrl)
             }
             catch (Exception e)
             {
-                _log.Info(nameof(ValidateHotwalletAsync), "Could not check api validity", $"{serviceUrl}");
+                _log.Info(nameof(ValidateHotwalletAsync), "Could not check api validity", $"{serviceUrl}", e);
```

Comment:
Your message is passed to the ```context``` parameter. This overload is used https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Log/MicrosoftLoggingBasedLogExtensions.cs#L247. Specify ```message``` and ```exception``` parameter names explicitly.